samples/Server/eslint.config.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  baseConfig,
+  baseGlobalIgnores,
+  globalConfig,
+  testFilesBaseConfig
+} from '../../common/config/ESLint/eslint.config';
+import { globalIgnores } from 'eslint/config';
+
+export default globalConfig([
+  baseConfig,
+  testFilesBaseConfig,
+  {
+    rules: {
+      'headers/header-format': 'off',
```

Comment:
Looks like some rules have been disabled compared to the previous configuration.

Reply:
I see whats going on here. Ill fix this up.